packages/cli/src/util/get-latest-version/index.ts

```diff
@@ -67,25 +69,26 @@ export default function getLatestVersion({
     }
   }
 
-  if (!cache || cache.expireAt < Date.now()) {
+  if (!cache || !cache.expireAt || cache.expireAt < Date.now()) {
     spawnWorker(
```

Comment:
**question (soft-blocking):** do we need `spawnWorker` to pass along the No Color env vars and CLI argument in some form to make sure the border of the box is not colored when color is disabled?

That PR was only recently merged in: https://github.com/vercel/vercel/pull/8826

Reply:
`get-latest-worker.js` no longer uses colors because it no longer has any external dependencies. There shouldn't be any instance where colorized text is rendered to stdio.